CliFx/Models/CommandSchema.cs

```diff
@@ -30,15 +30,21 @@ public partial class CommandSchema
         /// </summary>
         public IReadOnlyList<CommandOptionSchema> Options { get; }
 
+        /// <summary>
+        /// Command arguments.
+        /// </summary>
+        public IReadOnlyList<CommandArgumentSchema> Arguments { get; }
+
         /// <summary>
         /// Initializes an instance of <see cref="CommandSchema"/>.
         /// </summary>
-        public CommandSchema(Type? type, string? name, string? description, IReadOnlyList<CommandOptionSchema> options)
+        public CommandSchema(Type? type, string? name, string? description, IReadOnlyList<CommandOptionSchema> options, IReadOnlyList<CommandArgumentSchema> arguments)
```

Comment:
Just to be a bit more consistent with the input itself, let's put `Arguments` before `Options` (also in constructor)